scalajslib/worker/1/src/ScalaJSWorkerImpl.scala

```diff
@@ -80,11 +91,11 @@ class ScalaJSWorkerImpl extends mill.scalajslib.api.ScalaJSWorkerApi {
       .withSourceMapURI(java.net.URI.create(sourceMap.getFileName.toString))
     val logger = new ScalaConsoleLogger
     val mainInitializer = Option(main).map { cls =>
-      ModuleInitializer.mainMethodWithArgs(cls, "main")
+      ScalaJSModuleInitializer.mainMethodWithArgs(cls, "main")
```

Comment:
Since you're now supporting `ModuleInitializer` in the frontend, I think you should use the value set by the user even in `fastOpt`, not only in `fastLinkJS`.
What do you think?